openedx/features/survey_report/admin.py

```diff
@@ -80,4 +80,10 @@ def get_actions(self, request):
             del actions['delete_selected']
         return actions
 
+    def report_state(self, obj):
+        if obj.surveyreportupload_set.last().is_uploaded():
```

Comment:
since we are doing this to avoid changing the model at this stage, can we at least leave a comment explaining that?

Reply:
Added comment on the method.